sdk/core/azure-core/inc/azure/core/url.hpp

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Define an Uniform Resource Location (URL).
+ */
+
+#pragma once
+
+#include "azure/core/case_insensitive_containers.hpp"
+
+#include <cstdint>
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_set>
+
+namespace Azure { namespace Core {
```

Comment:
Please don't make file moves yet. Just make the namespace changes in the first pass, so things are easier to review.

Reply:
Url is inside Azure::Core::Http.  I would need to create Azure::Core namespace in Http file.

I am not moving Url.hpp from some place to other, I am moving out of the Azure::Core::Http namespace. No matter how I do it, you will see the entire Url moved to some place else.